packages/core/lib/commands/debug.js

```diff
@@ -30,18 +30,18 @@ const command = {
   },
   help: {
     usage:
-      "truffle debug [<transaction_hash>] [--fetch-external]" + OS.EOL +
-      "                             [--compile-tests|--compile-all|--compile-none]",
+      "truffle debug [<transaction_hash>] [(--fetch-external|-x)]" + OS.EOL +
+      "                             [(--compile-tests|--compile-all|--compile-none)]",
     options: [
       {
         option: "<transaction_hash>",
         description:
           "Transaction ID to use for debugging.  Mandatory if --fetch-external is passed."
       },
       {
-        option: "--fetch-external",
+        option: "(--fetch-external|-x)",
```

Comment:
Why are there parentheses here?  These are top-level parentheses, it's not grouping these within anything!

Reply:
Ok so maybe we don't need the parens here, I'm ok with that. I'm just trying to establish a standard. I think above that they are meant to group the option and its alias to say "hey, only one of the things inside the parens". You vote them gone completely? Also, spaces between options and the | or not? See the above screenshot.

Reply:
seems like no spaces is more common

Reply:
> Ok so maybe we don't need the parens here, I'm ok with that. I'm just trying to establish a standard. I think above that they are meant to group the option and its alias to say "hey, only one of the things inside the parens". You vote them gone completely? Also, spaces between options and the | or not? See the above screenshot.

No grouping is needed because there's no precedence inversion or ambiguity.  You only group when it would be incorrect otherwise.